moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -594,7 +589,7 @@ std::pair<bool, KinematicState> Servo::smoothHalt(const KinematicState& halt_sta
 {
   bool stopped = false;
   auto target_state = halt_state;
-  const auto current_state = getCurrentRobotState();
+  KinematicState current_state = getCurrentRobotState();
```

Comment:
Same here, can this remain const or did something change in the filtering?

Reply:
done